cmd/kubectl-testkube/commands/testworkflows/run.go

```diff
@@ -45,12 +46,13 @@ func NewRunTestWorkflowCmd() *cobra.Command {
 		format                   string
 		masks                    []string
 		tags                     map[string]string
+		selectors                []string
```

Comment:
Are these selectors or a single `AND` selector? In example, does `[]string{"a=b", "c=d"}` mean items that match both or either?

Reply:
Both, it prepares standard k8s selector, that we use to select test workflows, user can submit either --label a=b,c=d or --label a=b --label c=d

For either user can submit --label "a in (b,c)"

Reply:
By reading the comment - it means that it's `OR`, right? In example `--label="a in (b, c)" --label="a=d" --label="c=d"` will run workflows:
* have `a` equal to `b`, `c`, or `d`, or
* have `c` equal to `d`

Reply:
```--label="a=b" --label="c=d"``` means AND  -  a equall b and c equal d. it will  be stringified as selector ```a=b,c=d``` 
```--label="a in (b, c)"``` means OR - a equal b or a equal c
 ```--label="a in (b, c)" --label="a=b" --label="c=d"```  means (a equal b or a equal c) AND (a equal b) AND (c equal d)

https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
```The API currently supports two types of selectors: equality-based and set-based. A label selector can be made of multiple requirements which are comma-separated. In the case of multiple requirements, all must be satisfied so the comma separator acts as a logical AND (&&) operator.```

For expresssions
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement